web_src/css/repo.css

```diff
@@ -1714,6 +1714,18 @@ td .commit-summary {
 .file-view.markup {
   padding: 1em 2em;
 }
+
+.file-view.markup:has(.file-is-empty) {
+  padding: 0;
+}
+
+.file-is-empty {
+  padding: 1rem;
+  text-align: center;
+  font-size: 1rem !important; /* undo style from .markup parent */
+  line-height: var(--line-height-default) !important; /* undo style from .markup parent */
+}
```

Comment:
I strongly have objection for some tricky fine-tunes.

1. I do not think making long sentences center is more readable. We should avoid unnecessary "center" styles.
2. I do not think it's worth to "fine-tune" for markup, and I think it's also good to have different visual feedbacks (just like rendering markup vs rendering source code, the UI looks different)
3. There is also a similar template "filetoolarge"

Reply:
1. I prefer center text because it matches GitHub. For example large file:

<img width="1332" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/afed5f3c-f60a-41e8-82ac-96d40d7e82ec">

The center text makes it immediately obvious that this is not file content but a message.

2. I think consistency is important
3. I've been meaning to unify all similar messages, File too large, File empty, CSV too many lines etc, so this is the first step.